tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/asynchronous/AsyncClassLevelClient.java

```diff
@@ -41,31 +41,19 @@
 @Asynchronous
 public class AsyncClassLevelClient {
 
-
     /**
-     * service 1 second in normal operation.
-     * @return the result as a Future
-     * @throws InterruptedException the interrupted exception
+     * Service an operation until waitCondition is completed or 1 second timeout.
```

Comment:
This should probably read the same as the `serviceCS` javadoc. Or perhaps we should just say `@see #serviceCS`.

Reply:
done